packages/react-core/src/components/Select/Select.tsx

```diff
@@ -13,13 +32,19 @@ export interface SelectProps extends MenuProps, OUIAProps {
   isOpen?: boolean;
   /** Single itemId for single select menus, or array of itemIds for multi select. You can also specify isSelected on the SelectOption. */
   selected?: any | any[];
-  /** Renderer for a custom select toggle. Forwards a ref to the toggle. */
-  toggle: (toggleRef: React.RefObject<any>) => React.ReactNode;
+  /** Select toggle. May be a direct ReactNode combined with the toggleRef property, or a renderer for a custom select toggle which forwards a ref to the toggle. */
```

Comment:
The description is not quite right.  Aren't both option a custom toggle?  I think we want to be explicit that if you do not use a renderer, you need to provide a ref.  I wonder if we should group the props somehow. Maybe an object for the react node toggle and the ref.  then the toggle prop could be either that or the renderer?

Reply:
Since there's no default I thought that "custom" was a bit redundant to state. For the required ref, I tried to convey that in `May be a direct ReactNode combined with the toggleRef property [...]` but let me try to reword that better.

Reply:
@tlabaj Do the descriptions make more sense now?